src/main/java/uk/gov/pay/api/exception/ConnectorResponseErrorException.java

```diff
@@ -4,23 +4,19 @@
 
 class ConnectorResponseErrorException extends RuntimeException {
 
-    private int errorStatus;
-    private String errorBody;
+    private int status;
 
     ConnectorResponseErrorException(Response response) {
-        this.errorStatus = response.getStatus();
-        this.errorBody = response.readEntity(String.class);
+        super(response.toString());
```

Comment:
`toString()` -> Isn't this just the ObjectID of Response class? And don't we always send a error body from connector ? (so that we could readEntity())